gradle/dependencies.gradle

```diff
@@ -22,29 +22,37 @@ ext {
   libs = [:]
   
   // Enabled by default when commands like `testAll` are invoked
-  defaultScalaVersions = [ '2.11', '2.12' ]
+  defaultScalaVersions = [ '2.11', '2.12', '2.13' ]
```

Comment:
I suggest we add this line after 2.13 is released.

Reply:
Sure @ijuma, I agree on all accounts. 

Scala **2.13.0-M5** will be stable enough; (a way to early) PR was inspired by this JIRA ticket: https://issues.apache.org/jira/browse/KAFKA-7197 ***Release a milestone build for Scala 2.13.0 M3***

Reply:
Resolving conversation (see git commit changes).